<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="deckListWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>221</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="deckLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Decks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextCardButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Next Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevCardButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Previous Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotItButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Got It!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Flip Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartDeckButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Restart Deck</string>
    </property>
   </widget>
   <widget class="QLabel" name="topLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Upper Text</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bottomLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lower Text</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="shuffleButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Shuffle Deck</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="defFirstBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>480</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Definition First</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCreate_New_Deck"/>
    <addaction name="actionImport_Deck_CSV"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_New_Deck">
   <property name="text">
    <string>Create New Deck</string>
   </property>
  </action>
  <action name="actionImport_Deck_CSV">
   <property name="text">
    <string>Import Deck CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
